internal/engine/engine_test.go

```diff
@@ -53,15 +54,34 @@ func (suite *EngineSuite) EqualTables(expected, got table.Table) {
 	suite.NoError(err)
 	gotIt, err := got.Rows()
 	suite.NoError(err)
+
+	var expectedRows []table.Row
+	var expectedErr error
+	var gotRows []table.Row
+	var gotErr error
+
+	for {
```

Comment:
Order independent comparison.
First for loop collects all expected rows in memory, second loop all actual rows.
after that, line 82-84, it is checked that all actual rows are expected.
This works, because in line 80 there is a check that the amount of expected and actual rows match.

Reply:
Can this be added there?

Reply:
Probably not worth, as this will change as soon as we require sorted comparison for some test cases, i.e. as soon as we support `ORDER BY`.